examples/baytownBucks.js

```diff
@@ -1,26 +1,31 @@
-import { makeMint } from '@agoric/ertp/core/mint';
+import produceIssuer from '@agoric/ertp';
 
 // Let's create a community currency and make a payment for Alice.
 // This example uses ERTP but does not use Zoe, since no offers are
 // being made.
 
-// First, let's create a mint
-const baytownBucksMint = makeMint('BaytownBucks');
+// First, let's create an issuer
+export const baytownBucksBundle = produceIssuer('BaytownBucks');
 
-// Now let's get the assay from the mint
-// Note: exported for testing purposes
-export const baytownBucksAssay = baytownBucksMint.getAssay();
+// Now let's get the mint from the issuer
+const baytownBucksMint = baytownBucksBundle.mint;
 
-// Let's give ourselves a easy way of labeling a number as baytownBucks.
+// Let's give ourselves a way to create amounts of baytownBucks.
 // Note: exported for testing purposes
-export const baytownBucks = baytownBucksAssay.makeUnits;
+export const baytownBucks = baytownBucksBundle.amountMath;
```

Comment:
```suggestion
export const baytownBucks = baytownBucksBundle.amountMath.make;
```
I think the entire amountMath shouldn't be called `baytownBucks` - just the `make` part

Reply:
done